misc/dump-ast.py

```diff
@@ -28,20 +28,14 @@ def main() -> None:
     parser = argparse.ArgumentParser(
         description="Parse source files and print the abstract syntax tree (AST)."
     )
-    parser.add_argument("--py2", action="store_true", help="parse FILEs as Python 2")
```

Comment:
Note that this is our internal tool, so we can remove this option, end users are not affected.